### PR TITLE
Fixed logger stacktrace

### LINKED ISSUE #34: Error Logging Doesn't Report Actual Origin
**Describe the bug**
When calling things like logging.Error or logging.Panic, the location printed is the logging package function, not the actual caller of the function

**To Reproduce**
Call a logging.Error and see the file and line its printed on

**Expected behavior**
The file and line printed should be the actual code where there is an issue

### PR DESCRIPTION
**Pull request by Smartinez3324**
This closes #34 

# Description
Added skipped a step in the stack trace so that it just goes to the next step, skips the logger file.

# How Has This Been Tested?
Added logging messages in multiple locations around the project and checked the logging messages, they all seem to be where they are supposed to be.

# Checklist:
- [x] New functionality is documented in the necessary spots (i.e new functions documented in the header)
- [x] Unit tests cover any new functionality or edge cases that the PR was meant to resolve (if applicable) 
- [x] The CI checks are passing
- [x] I reviewed my own code in the GitHub diff and am sure that each change is intentional

### CODE REVIEW

**Comment by Smartinez3324** [comment]
the pr for 1 line 🙌

**Review by AarC10 — CHANGES_REQUESTED** [review]
Fix static analysis. But looks good.

**Review by AarC10 — APPROVED** [review]

File: cmd/gsw_service.go [hunk 1]
```diff
@@ -148,7 +148,12 @@ func main() {
 	if profilingPort != 0 {
 		go func() {
 			logger.Info(fmt.Sprintf("Running pprof server at localhost:%d", profilingPort))
-			http.ListenAndServe(fmt.Sprintf("localhost:%d", profilingPort), nil)
+			err := http.ListenAndServe(fmt.Sprintf("localhost:%d", profilingPort), nil)
+
+			if err != nil {
+				logger.Warn("Unable to listen and serve")
```
Comment (apenza) [inline cmd/gsw_service.go:154]: It would be nice if we logged the error message as well. But otherwise looks good!

**Review by apenza — APPROVED** [review]
